test/e2e/memcached_test.go

```diff
@@ -27,7 +27,11 @@ const (
 )
 
 func TestMemcached(t *testing.T) {
-	os.Chdir(os.Getenv("GOPATH") + "/src/github.com/example-inc")
+	gopath, ok := os.LookupEnv("GOPATH")
+	if !ok {
+		t.Fatalf("GOPATH not set")
+	}
+	os.Chdir(gopath + "/src/github.com/example-inc")
```

Comment:
Use [path.Join()](https://golang.org/pkg/path/#Join) for filepaths? Here and on line 44